test/unit/spec/EnrollWebauthn_spec.js

```diff
@@ -88,17 +88,19 @@ function (Okta,
       navigator.credentials = { create: function () {} };
     }
 
-    function mockWebauthnSuccessRegistration () {
+    function mockWebauthnSuccessRegistration (doNotResolve) {
       mockWebauthn();
       spyOn(webauthn, 'isNewApiAvailable').and.returnValue(true);
       spyOn(navigator.credentials, 'create').and.callFake(function () {
         var deferred = Q.defer();
-        deferred.resolve({
-          response: {
-            attestationObject: CryptoUtil.strToBin(testAttestationObject),
-            clientDataJSON: CryptoUtil.strToBin(testClientData)
-          }
-        });
+        if (!doNotResolve) {
```

Comment:
2 comments here:
1. can we make this not a "double negative" -> if NOT doNotResolve... maybe the arg can be named something like `resolvePromise` or `successfullyResolvePromise`?
2. If it does not resolve, then we are timing out? Not sure how that is handled in the test...

Reply:
we will be in waiting state. More like faking the state where we see the webauthn prompt but user has not yet authenticated via touchid/u2f key